src/basics.md

```diff
@@ -1265,6 +1266,32 @@ fn main() {
     println!("Time elapsed in expensive_function() is: {:?}", duration);
 }
 ```
+[ex-convert-datetime-timezone]: #ex-convert-datetime-timezone
+<a name="ex-convert-datetime-timezone"></a>
+## Converting a local time to UTC time and vice versa
+[![chrono-badge]][chrono] [![cat-date-and-time-badge]][cat-date-and-time]
+
+Gets the local time and displays it using [`offset::Local::now`] and then converts it to the UTC format using the [`DateTime::from_utc`] struct method. A time is then converted using the [`offset::FixedOffset`] struct and the UTC time is the converted to UTC+8 and UTC-.
+
+```rust
+extern crate chrono;
+
+use chrono::{DateTime, FixedOffset, Local, Utc};
+
+fn main() {
+    let local_time: DateTime<Local> = Local::now();
+    let utc_time: DateTime<Utc> = DateTime::<Utc>::from_utc(local_time.naive_utc(), Utc);
+    let china_timezone: FixedOffset = FixedOffset::east(28_800);
+    let rio_timezone: FixedOffset = FixedOffset::west(7_200);
+    println!("Local time now is {}", local_time);
```

Comment:
It would be cool to show what is the local TZ offset (as well as printout the other TZ offsets)
But I'll leave it up to you.